app/gui/src/ide/integration.rs

```diff
@@ -38,15 +38,20 @@ impl Model {
         if let Some(project_model) = self.controller.current_project() {
             // We know the name of new project before it loads. We set it right now to avoid
             // displaying placeholder on the scene during loading.
-            self.view.graph().model.breadcrumbs.project_name(project_model.name().to_string());
+            self.view
+                .project_view()
+                .graph()
+                .model
+                .breadcrumbs
+                .project_name(project_model.name().to_string());
```

Comment:
Please refactor the codebase (here and in other places). Prefer multiple variables over one expression divided into multiple lines (this is also described in our style doc). I'll not be adding similar comments in other places of the PRs - please browse all of them and fix similar places.

Reply:
Done. There is only one multiline expression in this diff I believe, but it's impossible to get rid of it without using crazy short variable names.